backend/tests/apps/core/api/status_test.py

```
@@ -0,0 +1,42 @@
+import pytest
+from django.conf import settings
+from django.test import RequestFactory
+
+from apps.core.api.status import get_status
+
+
+class TestGetStatusFunction:
+    """Tests for the get_status API function."""
+
+    @pytest.fixture(autouse=True)
+    def setup_client(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)